proto/substrait/algebra.proto

```diff
@@ -228,6 +228,71 @@ message AggregateRel {
   }
 }
 
+// The Window relational operate on a group of rows, referred to as a window, and calculate a return value for each row based on the group of rows.
+message WindowRel {
```

Comment:
I don't think should be called WindowRel. How about ConsistentPartitionWindowRel or something else that clearly describes the intention.

Reply:
`ConsistentPartitionWindowRel ` is ok to me.